.travis.yml

```diff
@@ -4,3 +4,6 @@ node_js:
   - 'iojs'
   - '0.12'
   - '0.10'
+before_install:
+  - "npm config set spin false"
+  - "npm install -g npm@^2"
```

Comment:
This is usually to fix npm on node 0.10, but tests passed before this, so I don't think it's strictly needed.